### DISCUSSION TITLE
Maven deploy and 422 Unprocessable Entity error

### DISCUSSION THREAD

**Discussion by sblantipodi** [community/community#25273] [Actions]
<p>Hi guys,<br>
I am trying to maven deploy my jar files using maven deploy command,<br>
when I try to maven deploy it says:</p>
<blockquote>
<p>FireflyLuciferin: Failed to deploy artifacts: Could not transfer artifact org.dpsoftware:FireflyLuciferin:jar:1.5.2 from/to github (<a href="https://maven.pkg.github.com/sblantipodi/firefly_luciferin" rel="noopener nofollow ugc">https://maven.pkg.github.com/sblantipodi/firefly_luciferin</a>): Transfer failed for <a href="https://maven.pkg.github.com/sblantipodi/firefly_luciferin/org/dpsoftware/FireflyLuciferin/1.5.2/FireflyLuciferin-1.5.2.jar" rel="noopener nofollow ugc">https://maven.pkg.github.com/sblantipodi/firefly_luciferin/org/dpsoftware/FireflyLuciferin/1.5.2/FireflyLuciferin-1.5.2.jar</a> 422 Unprocessable Entity -&gt; [Help 1]</p>
</blockquote>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/sblantipodi/firefly_luciferin/runs/1294716862?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/4722227/181090037-4c40a3d3-43ce-422a-8b2e-f0a63c824a35.jpeg" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/sblantipodi/firefly_luciferin/runs/1294716862?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">sblantipodi/firefly_luciferin</a></h3>

<p>Very fast Java Screen Capture software designed for Glow Worm Luciferin firmware - sblantipodi/firefly_luciferin</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>maven package works well and the artifact works well.</p>
<p>any idea on how to solve?<br>
thanks.</p>

**Comment by sblantipodi** [comment, marked as answer]
<p>fixed it using an lowercase artifactid.<br>
artifactid must be lowercase!</p>

**Reply by krachynski** [reply]
Has anybody ever explained why this is the case? The Maven documentation says that artifactId _should_ be lowercase, not must.

**Comment by minhtranes** [comment]
<p>I encountered the same issue. It becauses of the artifactId with dot (.). Remove it fixed my problem</p>
